wp-content/themes/pub/wporg-learn-2024/inc/block-config.php

```diff
@@ -557,3 +559,78 @@ function inject_other_filters( $key ) {
 		printf( '<input type="hidden" name="s" value="%s" />', esc_attr( $wp_query->query['s'] ) );
 	}
 }
+
+/**
+ * Filter the block data for the student course list.
+ *
+ * @param array $parsed_block The parsed block data.
+ * @return array The updated block data.
+ */
+function filter_student_course_list( $parsed_block ) {
```

Comment:
I think we could improve the docs here
```suggestion
/**
 * Modifies the courses query for the My Courses page, by adding courses to be excluded.
 *
 * @param array $parsed_block The parsed block data.
 * @return array The updated block data.
 */
function filter_student_course_list( $parsed_block ) {
```

Reply:
bec4e5c